tests/functional/search-auth.js

```diff
@@ -198,6 +198,34 @@ registerSuite("Search (auth)", {
         .end();
     },
 
+    "Label have correct URL after search without results"() {
+      // load up a garbage search, so we can more easily detect when
+      // the search values we want are loaded.
+      return FunctionalHelpers.openPage(
+        this,
+        url(
+          "/issues"
+        ),
+        ".grid"
+      )
+        .findByCssSelector("#js-SearchForm-input")
+        .click()
+        .type("noExpectedResult123")
+        .submit()
+        .end()
+        .findByCssSelector("#x-search-bar")
+        .submit()
+        .end()
+        .sleep(2000)
+        .findDisplayedByCssSelector(".label-browser-android")
+        .click()
+        .end()
+        .getCurrentUrl()
+        .then(function(url) {
+          assert(url.includes("q=label%3Abrowser-android"));
```

Comment:
Can you add a description as a 2nd argument to `assert`?

Reply:
(oh weird, i already wrote this comment...) 👻

Reply:
fixed above.